### PR TITLE
Authorization v2.5.1

### LINKED ISSUE #371: Check authorization on endpoints
## overview
with all recent updates from v2.5.0 and v2.5.1 it is important to not allow users to do things they should not be allowed to do.

examples
- get a secret code from the api
- leave a group impersonating another user
- any impersonation should be double checked
- get vote score from api [can be seen in cycles]

### PR DESCRIPTION
**Pull request by diegoalzate**
closes #371 

## overview
- hides secret and voteScore through api
- makes user authorization more explicit

### CODE REVIEW

File: src/handlers/options.ts [hunk 1]
```diff
@@ -12,17 +12,11 @@ export function getOptionHandler(dbPool: PostgresJsDatabase<typeof db>) {
...
-        id: db.questionOptions.id,
-        userId: db.questionOptions.userId,
-        registrationId: db.questionOptions.registrationId,
-        questionId: db.questionOptions.questionId,
-        optionTitle: db.questionOptions.optionTitle,
-        optionSubTitle: db.questionOptions.optionSubTitle,
-        accepted: db.questionOptions.accepted,
-        createdAt: db.questionOptions.createdAt,
-        updatedAt: db.questionOptions.updatedAt,
+        ...rest,
```
Comment (MartinBenediktBusch) [inline src/handlers/options.ts:19]: Vote score should be excluded as part of the option handler.

**Review by MartinBenediktBusch — CHANGES_REQUESTED** [review]
I think the vote score will be exposed by the getOptionHandler again with this PR.

File: src/handlers/options.ts (hunk 1 above)
Comment (diegoalzate) [reply src/handlers/options.ts:19]: I believe it is being excluded

**Review by MartinBenediktBusch — APPROVED** [review]
lgtm now. Missed the exclusion through const { voteScore, ...rest } = getTableColumns(db.questionOptions);

**Pull request merged**